packages/shared/src/components/modals/SquadsBetaModal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement, useState } from 'react';
+import SquadsBackgroundSvg from '../../svg/SquadsBackground';
+import { Button } from '../buttons/Button';
+import { Justify } from '../utilities';
+import { Modal } from './common/Modal';
+import {
+  ConfirmationButtons,
+  ConfirmationDescription,
+  ConfirmationHeading,
+  ConfirmationModal,
+} from './ConfirmationModal';
+
+export type SquadsBetaModalProps = {
+  onRequestClose: () => void;
+  onNext: () => void;
+};
+
+type ConfirmationProps = {
+  onRequestClose: () => void;
+  onContinue: () => void;
+};
+
+function Confirmation({
+  onRequestClose,
+  onContinue,
+}: ConfirmationProps): ReactElement {
+  return (
+    <ConfirmationModal isOpen>
+      <ConfirmationHeading>Quit the process?</ConfirmationHeading>
+      <ConfirmationDescription>
+        <p>
+          Learning is more powerful together. Are you sure you want to quit the
+          process?
+        </p>
+        <p>p.s you can create a new squad from the left sidebar</p>
+      </ConfirmationDescription>
+      <ConfirmationButtons>
+        <Button className="btn-secondary" onClick={onRequestClose}>
+          Cancel
+        </Button>
+        <Button className="btn-primary-ketchup" onClick={onContinue}>
+          Continue
+        </Button>
+      </ConfirmationButtons>
+    </ConfirmationModal>
+  );
+}
+
+function SquadsBetaModal({
+  onRequestClose,
+  onNext,
+}: SquadsBetaModalProps): ReactElement {
+  const [showConfirmation, setShowConfirmation] = useState(false);
+
+  return (
+    <>
+      <Modal isOpen kind={Modal.Kind.FixedCenter} size={Modal.Size.Small}>
```

Comment:
Another alternative is to incorporate this screen in the NewSquadModal, that way you will not see the flicker between modals.
It might also help with the abstraction on the sidebar part.
We have an example in the feedFilterOnboarding. 

Would love to hear what you think of that?

Reply:
I thought about that, but the reason I went this way was to separate this since it seemed like a beta-only screen. I am also open to moving it inside if you like.

Reply:
If it doesn't have any UX issues I'm fine with this.
Just remembered it was an issue before, so we'll need to verify.

Reply:
The reason for the flicker is that the new squad modal is loaded asynchronously after the beta screen, I can fix that